src/js/modules/rpc/server.ts

```diff
@@ -31,10 +31,52 @@ import errors, { DisableResponseCode } from "./errors";
 import { Logger } from "winston";
 import Logging from "../utilities/Logging";
 import requireNative from "./require-native";
+import { isNtpEqual } from "../utilities/domain";
+import { find } from "../utilities/Map";
+
+function groupByTopic(
+  processBatchReplyItems: ProcessBatchReplyItem[]
+): ProcessBatchReplyItem[] {
+  interface Map {
```

Comment:
Hard to understand, there is a new type called `Map` ? Isn't there a nodejs type with the same name?

Reply:
Change the type name